AssemblerLib/LocalDataInitializer.h

```diff
@@ -240,18 +242,19 @@ class LocalDataInitializer
 
     /// Sets the provided data_ptr to the newly created local assembler data.
     void operator()(
-            const MeshLib::Element& e,
+            std::size_t const element_id,
```

Comment:
I remember in the docs of the old LocalAssemblerBuilder, there was a comment
> \attention The index \c id is not necesserily the mesh item's id.

I don't know, when they can be different. Let's discuss tomorrow.